drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -126,13 +126,24 @@ def self.options_for_type(pick_list_type, user:, relation_id: nil)
         Hmis::File.all_available_tags.map do |tag|
           {
             code: tag.id,
-            label: "#{tag.name} (included: #{tag.included_info})",
+            label: tag.name,
             group_code: tag.group,
             group_label: tag.group,
+            secondary_label: "(includes: #{tag.included_info})",
           }
         end.
           compact.
           sort_by { |obj| [obj[:group_label] + obj[:label]].join(' ') }
+      when 'CLIENT_ENROLLMENTS'
+        client = Hmis::Hud::Client.viewable_by(user).find_by(id: relation_id)
+        return [] unless client.present?
+
+        client.enrollments.sort_by_option(:most_recent).map do |enrollment|
+          {
+            code: enrollment.id,
+            label: "#{enrollment.project.project_name} (#{[enrollment.entry_date.strfdate('mm/dd/yyyy'), enrollment.exit_date&.strfdate('mm/dd/yyyy') || 'ongoing'].join(' - ')})",
```

Comment:
I'm getting an error: `undefined method `strfdate' for Wed, 30 Dec 2020:Date↵Did you mean? strftime`

Reply:
Oops, I think I did mean that. Fixed

Reply:
And the formatting string is incorrect
<img width="462" alt="Screen Shot 2023-03-24 at 10 06 08 AM" src="https://user-images.githubusercontent.com/5333982/227543219-f51ca08f-5779-4c9e-b3ae-660c44102c43.png">

Reply:
🤦‍♂️ Fixed